pydecred/rpc.py

```diff
@@ -60,6 +60,176 @@ def getBlockchainInfo(self):
+        return with an error stating the address index has not yet been built.
+        Similarly, until the address index has caught up with the current best
+        height, all requests will return an error response in order to avoid
+        serving stale data.
+
+        Args:
+            address (string): The Decred address to search for
+            verbose (int): Specifies the transaction is returned as a JSON
+                object instead of hex-encoded string.
+            skip (int): The number of leading transactions to leave out of the
+                final response.
+            count (int): The maximum number of transactions to return.
+            vinextra (int): Specify that extra data from previous output will be
+                returned in vin.
+            reverse (bool): Specifies that the transactions should be returned
+                in reverse chronological order.
+            filteraddrs (list(string)): Only inputs or outputs with matching
+                address will be returned.
+
+        Returns:
+            list(SearchRawTransactionResult): The SearchRawTransactionResults.
+        """
+        return [
+            SearchRawTransactionsResult.parse(rawTx)
+            for rawTx in self.call(
+                "searchrawtransactions",
+                address,
+                verbose,
+                skip,
+                count,
+                vinextra,
+                reverse,
+                filteraddrs,
+            )
+        ]
+
+    def sendRawTransaction(self, hextx, allowhighfees=False):
+        """
+        Submits the serialized, hex-encoded transaction to the local peer and
+        relays it to the network.
+
+        Args:
+            hextx (string): Serialized, hex-encoded signed transaction.
+            allowhighfees (bool): Whether or not to allow insanely high fees
+                (dcrd does not yet implement this parameter, so it has no effect).
+
+        Returns:
+            string: The hash of the transaction.
+        """
+        return self.call("sendrawtransaction", hextx, allowhighfees)
+
+    def setgenerate(self, generate, genproclimit=-1):
+        """
+        Set the server to generate coins (mine) or not.
+
+        Args:
+            generate (bool): Use True to enable generation, False to disable it.
+            genproclimit (int): The number of processors (cores) to limit
+                generation to or -1 for default.
+        """
+        self.call("setgenerate", generate, genproclimit)
+
+    def stop(self):
+        """
+        Shutdown dcrd.
+
+        Returns:
+            string: 'dcrd stopping.'
+        """
+        return self.call("stop")
+
+    def submitBlock(self, hexblock, options={}):
+        """
+        Attempts to submit a new serialized, hex-encoded block to the network.
+
+        Args:
+            hexblock (string): Serialized, hex-encoded block.
+            options (dict[workid]string): This parameter is currently ignored.
```

Comment:
The type is a little confusing here. Maybe just omit the type since we're not using it anyway.